src/modules/common/Logger.ts

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
+
+    Logging class for Stoa
+
+    Define the output format of Log and support console output
+    and file writing, depending on the 'NODE_ENV'.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import winston from "winston";
+import { format, transports } from "winston";
+import appRoot from "app-root-path";
+
+const { combine, timestamp, label, printf } = format;
+const logFormat = printf(({ level, message, label, timestamp }) => {
+  return `[${label}] ${timestamp} ${level} ${message}`;
+});
+
+const options = {
+    // write log file options
+    file: {
+      filename: `${appRoot}/logs/Stoa.log`, // TODO: move to config
+      handleExceptions: true,
+      json: false,
+      maxsize: 10485760, // 10MB
+      maxFiles: 10,
```

Comment:
The filename doesn't change depending on the date, do I need this `maxFiles`?
What happens if the file size is more than 10 MByte?
If the file size is more than 10 MByte and the log is not recorded, it would be nice to have the file renamed every day.

Reply:
There is a good library related to this.
Do you want to be managed like Web log?

Reply:
I think it'll be better to setup something like `logrotate` so that is handled outside of the `stoa` node.

Reply:
I think Stoa can run outside of Linux.